virtio-devices/src/vhost_user/fs.rs

```diff
@@ -38,7 +38,7 @@ use vm_migration::{
 use vmm_sys_util::eventfd::EventFd;
 
 const NUM_QUEUE_OFFSET: usize = 1;
-const DEFAULT_QUEUE_NUMBER: usize = 2;
+const DEFAULT_QUEUE_NUMBER: usize = 1;
```

Comment:
I don't think this is the right thing to do since there are at least 2 queues (1 `hiprio` and 1 `request`) based on the spec https://github.com/oasis-tcs/virtio-spec/blob/virtio-1.3/virtio-fs.tex

Reply:
Yes, I can see the wording was clarified vs the 1.2 spec. Also I mistook the "num_queues" parameter for `FsConfig` as actually the number of request queues.